src/components/with-checkout.js

```diff
@@ -141,7 +141,7 @@ export function withCheckout (WrappedComponent) {
           { currentStep === 5 && <MiniPlaceOrder
             convertToOrder={this.wrappedRef.current.convertToOrder}
             total={cart.total}
-            isValidOrder={this.wrappedRef.current.isValidOrder(cart, order)}
+            isValidOrder={this.wrappedRef.current.isValidOrder && this.wrappedRef.current.isValidOrder(cart, order)}
```

Comment:
This fixes a console error caused by the ref being updated on line 150, after this line evaluates.